app/assets/stylesheets/common/base/topic-post.scss

```diff
@@ -737,8 +737,32 @@ aside.quote {
   opacity: 0.4;
 }
 
-.fk-d-menu[data-identifier="post-text-selection-toolbar"] {
+// disabling user select on the whole page while selecting a post text
+body.-disable-global-text-selection * {
   @include user-select(none);
+
+  // specifically enable text selection on the post with current selection
+  .allow-post-text-selection * {
+    @include user-select(text !important);
+  }
+}
+
+html.mobile-device {
```

Comment:
This doesn't seem related to viewport-width, so I don't think we should be using `.mobile-device` here?

Reply:
how do I target android?